### DISCUSSION TITLE
Reusable Taskfile templates

### DISCUSSION THREAD

**Discussion by jfx** [go-task/task#989] [Show and tell]
Hello,
I create a repository of reusable Taskfile templates for a quick and easy way to set up common tasks. Additionally, the installation process is quite simple and it makes it easier for others to use the templates. Every new templates is of course welcome.
I plan to define a contribution guide with some minimal coding rules and deploy a documentation site to easily navigate through the task descriptions.
What do you think of the project?
Any suggestions, improvements, or contributions are welcome!
Is it a good idea to have this repository in the go-task organization?
The repository is here: [https://gitlab.com/op_so/task/task-templates](https://gitlab.com/op_so/task/task-templates)

**Comment by andreynering** [comment]
Hi @jfx, 👋 

Sounds interesting, but I think it can continue to be a separate community repository.

Once it's more complete and have the mentioned website, we can consider adding a link to it from https://taskfile.dev.

**Reply by jfx** [reply]
Thanks for your answer, I shall notify you when the project will be more advanced. Thanks again for your investment in the Task project.

**Comment by jfx** [comment]
Hello, 
It's been a long road, but we're finally here! 
The repository of task templates is: [https://gitlab.com/op_so/task/task-templates](https://gitlab.com/op_so/task/task-templates)
And the documentation: [https://task-templates.pages.dev/](https://task-templates.pages.dev/)

Contributions are welcome to improve existing templates or create new ones !
